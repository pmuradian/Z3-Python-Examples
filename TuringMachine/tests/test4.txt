3 3 5
a b c
i s f
i
f
c i c i 1
a i b i 1
c i c s -1
b s a s -1
c s c f 0
10
c a a a c a a a a a