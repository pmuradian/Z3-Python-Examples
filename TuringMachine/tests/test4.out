YES

c|i a|  a|  a|  c|  a|  a|  a|  a|  a|
c|  a|i a|  a|  c|  a|  a|  a|  a|  a|
c|  b|  a|i a|  c|  a|  a|  a|  a|  a|
c|  b|  b|  a|i c|  a|  a|  a|  a|  a|
c|  b|  b|  b|  c|i a|  a|  a|  a|  a|
c|  b|  b|  b|s c|  a|  a|  a|  a|  a|
c|  b|  b|s a|  c|  a|  a|  a|  a|  a|
c|  b|s a|  a|  c|  a|  a|  a|  a|  a|
c|s a|  a|  a|  c|  a|  a|  a|  a|  a|
c|f a|  a|  a|  c|  a|  a|  a|  a|  a|
NO